protocol/client.go

```diff
@@ -77,7 +78,7 @@ func (c *CollectedClientData) Verify(storedChallenge string, ceremony CeremonyTy
 	// passed to the get() call.
 
 	challenge := c.Challenge
-	if 0 != strings.Compare(storedChallenge, challenge) {
+	if subtle.ConstantTimeCompare([]byte(storedChallenge), []byte(challenge)) == 1 {
```

Comment:
This should be `!= 1` not `== 1`, it returns 1 on a match.

Reply:
Ah yep, glad you spotted this, wish I did! Completely forgot there was a significant API difference between the two methods.